cumulusci.yml

```diff
@@ -48,69 +48,76 @@ tasks:
                 TDTM_Global_API.setTdtmConfig( TDTM_Global_API.getTdtmConfig() );
 
     execute_qa_apex:
-        description: Runs Apex setup for QA orgs.
+        description: Runs anonymous apex for qa to configure settings.
         class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
         options:
             path: scripts/setup.cls
             apex: qaSetup();
 
     retrieve_config_dev:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/dev
+        description: Retrieves the current changes in the scratch org into unpackaged/config/dev.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/dev
             namespace_tokenize: $project_config.project__package__namespace
 
     retrieve_config_qa:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/qa
+        description: Retrieves the current changes in the scratch org into unpackaged/config/qa.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/qa
             namespace_tokenize: $project_config.project__package__namespace
 
     retrieve_config_trial:
-        description: Retrieves the current changes in the scratch org into unpackaged/config/trial
+        description: Retrieves the current changes in the scratch org into unpackaged/config/trial.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
     robot:
+        # description: Runs a Robot Framework test from a .robot file.
         options:
             suites: robot/EDA/tests
             options:
                 outputdir: robot/EDA/results
 
     robot_libdoc:
+        # description: Generates documentation for project keyword files.
         options:
             path: robot/EDA/resources/EDA.py,robot/EDA/resources/EDA.robot
             output: robot/EDA/doc/Keywords.html
 
     update_admin_profile:
+        description: Retrieves, edits, and redeploys the System Administrator profile with full FLS perms for all objects/fields. Also sets record type visibilities.
+        group: Custom Tasks
         class_path: tasks.salesforce.UpdateAdminProfile
```

Comment:
Also, this isn't a custom task, I don't think - should we actually be overriding the description here? I think we're just overriding the default behavior in tasks/salesforce.py

Reply:
@jstvz too.

Reply:
Yeah, my view is that if any task depends on a python class in `tasks/` then it's a custom task and should be highlighted as such. I think it's helpful to keep that in mind when doing any troubleshooting or debugging.